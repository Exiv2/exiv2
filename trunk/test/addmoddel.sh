#!/bin/bash
# Test driver to run the addmoddel sample program
source ./functions.source

(	cd "$testdir"

	copyTestFile      exiv2-empty.jpg
	runTest addmoddel exiv2-empty.jpg
	runTest exiv2 -pv exiv2-empty.jpg

) > $results

reportTest

# That's all Folks!
##