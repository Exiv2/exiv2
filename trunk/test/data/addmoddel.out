Added a few tags the quick way.
Added key "Exif.Photo.DateTimeOriginal", value "1999:12:31 23:59:59"
Added key "Exif.Image.PrimaryChromaticities", value "1/2 1/3 2/3 3/4"
Modified key "Exif.Photo.DateTimeOriginal", new value "2000:12:31 23:59:59"
Modified key "Exif.Image.PrimaryChromaticities", new value "1/2 1/3 88/77 3/4"
Deleted key "Exif.Image.PrimaryChromaticities"
0x0110 Image        Model                       Ascii       7  Test 1
0x0115 Image        SamplesPerPixel             Short       1  162
0x011a Image        XResolution                 SLong       1  -2
0x011b Image        YResolution                 SRational   1  -2/3
0x8769 Image        ExifTag                     Long        1  90
0x9003 Photo        DateTimeOriginal            Ascii      20  2000:12:31 23:59:59
